content/en/docs/setup/production-environment/container-runtimes.md

```diff
@@ -266,6 +297,14 @@ visit [MCR Deployment Guide](https://docs.mirantis.com/mcr/20.10/install.html).
 Check the systemd unit named `cri-docker.socket` to find out the path to the CRI
 socket.
 
+### Docker Engine and Mirantis Container Runtime
+
+{{< note >}}
+If using the Docker Engine these instructions assume that you are using the
+[`cri-dockerd`](https://github.com/Mirantis/cri-dockerd) adapter to integrate
+Docker Engine with Kubernetes.
+{{< /note >}}
+
```

Comment:
i think we can omit this addition given both sections `### Docker Engine` and `### Mirantis Container Runtime` already mention `cri-dockerd`?